docs/book/src/multiversion-tutorial/testdata/project/cmd/main.go

```diff
@@ -64,8 +64,6 @@ func init() {
 /*
  */
 func main() {
-	/*
-	 */
```

Comment:
Can you please revert it ?
It is used in the docs to render the code and has no relation with the purpose of this PR

Reply:
done